ReactWindows/ReactNative.Shared/UIManager/ThemedReactContext.cs

```diff
@@ -16,15 +17,31 @@ public class ThemedReactContext : ReactContext
         /// <param name="reactContext">The inner context.</param>
         public ThemedReactContext(ReactContext reactContext)
         {
-            InitializeWithInstance(reactContext.ReactInstance);
+            if (reactContext == null)
+            {
+                throw new ArgumentNullException(nameof(reactContext));   
+            }
+
             _reactContext = reactContext;
         }
 
         /// <summary>
+        /// Gets the instance of the <see cref="INativeModule"/> associated
+        /// with the <see cref="IReactInstance"/>.
+        /// </summary>
+        /// <typeparam name="T">Type of native module.</typeparam>
+        /// <returns>The native module instance.</returns>
+        public T GetNativeModule<T>()
```

Comment:
If any additional pass-through is required then it must be added explicitly on an as-needed basis...